### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by darshandamre**
Created home page for video library
currently it only has placeholder videos
also check for responsiveness

[Preview](https://video-library-git-home-page-darshandamre.vercel.app)

### CODE REVIEW

**Review by tushargoyal812 — COMMENTED** [review]
code is very clean and to the point.
liked how you structured files and folders.
variable naming is also very good.
good work!

File: src/components/Card/Card.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.video {
+  background-color: #d4d4d4;
```
Comment (TejasShekar) [inline src/components/Card/Card.css]: use variables

File: src/components/Card/Card.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  border-radius: 5px;
+  transition: 0.3s ease-in-out;
+}
+
+.video:hover {
+  background-color: #e5e5e5;
```
Comment (TejasShekar) [inline src/components/Card/Card.css]: use color variables here as well

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.header {
+  position: sticky;
+  top: 0;
+  background: var(--zinc);
```
Comment (TejasShekar) [inline src/components/Header/Header.css:4]: nice color name !

**Review by TejasShekar — APPROVED** [review]
Code looks well structured. Great !

File: src/components/Card/Card.css (hunk 1 above)
Comment (darshandamre) [reply src/components/Card/Card.css]: will update

File: src/components/Header/Header.css (hunk 3 above)
Comment (darshandamre) [reply src/components/Header/Header.css:4]: thanks to tailwindcss

**Pull request merged**